guides/common/attributes-base.adoc

```diff
@@ -29,6 +29,44 @@
 :RepoRHEL7ServerSatelliteServerPuppetVersion: rhel-7-server-satellite-6.3-puppet4-rpms
 :RepoRHEL7ServerSatelliteCapsulePuppetVersion: rhel-7-server-satellite-capsule-6.3-puppet4-rpms
 
+//Added by Marie for GA
+:SatelliteSub: Red Hat Satellite Infrastructure Subscription
+:RepoRHEL7Server: rhel-7-server-satellite
+:RepoRHEL8Server: rhel-8-for-x86_64-baseos-rpms 
+// We would probably need to mention also "rhel-8-for-x86_64-appstream-rpms"
```

Comment:
correct, also for EL8Server, I would call it `RepoRHEL8BaseOS` as there is no more "server" repo as such (there is no difference between server and workstation in el8).

Reply:
@evgeni , thank you! I agree! I've updated the name of the attribute for BaseOS and also added a new attribute for AppStream. This is done in the "attributes-base.adoc" file.